### PR TITLE
Ajaxify the rest

### PR DESCRIPTION
**Pull request by jt-is-coding**
I ajaxified the rest!!

### CODE REVIEW

File: app/views/likes/create.js.erb
Comment (jrzricardo) [inline app/views/likes/create.js.erb:1]: HOWDY

File: app/views/likes/_form.html.erb
Comment (jrzricardo) [inline app/views/likes/_form.html.erb:1]: nice insert

File: app/views/follow_requests/destroy.js.erb
Comment (jrzricardo) [inline app/views/follow_requests/destroy.js.erb:1]: clean

File: app/views/follow_requests/create.js.erb
Comment (jrzricardo) [inline app/views/follow_requests/create.js.erb:1]: love a good bonk

File: app/views/follow_requests/_form.html.erb
Comment (jrzricardo) [inline app/views/follow_requests/_form.html.erb:1]: sweet insert

File: app/views/follow_requests/_follow_unfollow.html.erb
Comment (jrzricardo) [inline app/views/follow_requests/_follow_unfollow.html.erb:1]: moving some stuff around i see

File: app/controllers/likes_controller.rb
Comment (jrzricardo) [inline app/controllers/likes_controller.rb:1]: the render template is unnecessary for it to work, but i respect your pursuit of clarity

File: app/controllers/follow_requests_controller.rb
Comment (jrzricardo) [inline app/controllers/follow_requests_controller.rb:1]: nice

File: app/controllers/follow_requests_controller.rb [hunk 1]
```diff
@@ -28,6 +28,10 @@ def create
       if @follow_request.save
         format.html { redirect_back fallback_location: root_url, notice: "Follow request was successfully created." }
         format.json { render :show, status: :created, location: @follow_request }
+
```
Comment (ahuynh3a) [inline app/controllers/follow_requests_controller.rb:31]: nitpik: please be mindful of unnecessary white spaces, delete them if if you can!

File: app/controllers/follow_requests_controller.rb [hunk 2]
```diff
@@ -54,6 +58,10 @@ def destroy
     respond_to do |format|
       format.html { redirect_back fallback_location: root_url, notice: "Follow request was successfully destroyed." }
       format.json { head :no_content }
+
```
Comment (ahuynh3a) [inline app/controllers/follow_requests_controller.rb:61]: same as above

File: app/views/likes/_form.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<%= form_with(model: like, class: "d-inline-block") do |form| %>
+<%= form_with(model: like, local: false, class: "d-inline-block") do |form| %>
```
Comment (ahuynh3a) [inline app/views/likes/_form.html.erb:1]: Great job being mindful of when to use local: false compared to remote: true

File: app/views/likes/create.js.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+console.log("HELLO!!!");
```
Comment (ahuynh3a) [inline app/views/likes/create.js.erb:1]: I love that you include console.log to see if the file is touched

**Review by ahuynh3a — APPROVED** [review]
Hi Jeremy! Great job, your code looks great, easy to follow, and works!

**Review by rezahusain — APPROVED** [review]
Not much comment as Annie and Ricardo have addressed everything that I noticed! Some general observations though - Love your usage of print statements. Helps a lot with debugging. Also using the longhand form of render is okay as long as you understand what the code is trying to do. Looks really good though overall.

File: app/views/likes/create.js.erb (hunk 4 above)
Comment (Cindellai) [reply app/views/likes/create.js.erb:1]: good job with error checking

File: app/views/likes/_form.html.erb
Comment (Cindellai) [reply app/views/likes/_form.html.erb:1]: good use of console log

File: app/views/follow_requests/_form.html.erb [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<%= form_with(model: follow_request, class: "d-inline-block") do |form| %>
+<%= form_with(model: follow_request, class: "d-inline-block", local: false) do |form| %>
```
Comment (Cindellai) [inline app/views/follow_requests/_form.html.erb:1]: good job using local false instead of remote true, I messed that up when i was doing it

**Review by Cindellai — APPROVED** [review]
good clean code!

**Pull request merged**